src/app/home/home.component.html

```diff
@@ -0,0 +1,40 @@
+
+<section id="carouselExampleIndicators" class="carousel slide" data-ride="carousel">
+    <ol class="carousel-indicators">
+        <li data-target="#carouselExampleIndicators" data-slide-to="0" class="active"></li>
```

Comment:
Why are we calling this `carouselExampleIndicators`?

Reply:
done